dashboard/test/ui/features/teacher_tools/rubrics/ai_evaluate_student_code.feature

```diff
@@ -140,20 +140,20 @@ Feature: Evaluate student code against rubrics using AI
     And I wait for the page to fully load
     And I wait until element "h1:contains(Getting Started with Your AI Teaching Assistant)" is visible
     And I click selector ".introjs-skipbutton" if it exists
-    And I wait until element ".congrats" is gone 
+    And I wait until element ".congrats" is gone
     #Then I verify progress in the header of the current page is "attempted_assessment" for level 2
     And element "#ui-floatingActionButton" is visible
 
     # Teacher views AI evaluation status in settings tab
     When I click selector "#ui-floatingActionButton"
     And I wait until element "#uitest-rubric-content" is visible
-    And element ".uitest-run-ai-assessment" is enabled
+    And I wait until element ".uitest-run-ai-assessment" is enabled
 
     # Teacher switches to Class Management tab
     When I click selector "button:contains('Class Data')"
     And I wait until element ".uitest-run-ai-assessment-all" is visible
-    And I wait until element "#ui-teacherFeedback" is enabled 
```

Comment:
@cearachew please take a minute to set your editor to remove trailing whitespace if possible. I'm not sure why lint isn't catching it, maybe it is because we don't have that kind of linting for .feature files.